english/disney/disney.md

```diff
@@ -0,0 +1,2 @@
+You realize that the small building you are looking at is Sleeping Beauty's Castle. Somehow you have been transported to Disneyland.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.